migration/src/main/resources/META-INF/rewrite/axon-jakarta.yml

```diff
@@ -0,0 +1,74 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.axonframework.axon.UpgradeAxonFramework_4_7_Jakarta
+displayName: Upgrade to Axonframework 4.7 Jakarta
+description: This is what you will want if you also upgrade to Spring 6, Spring Boot 3, or another framework that depends on jakarta packages.
```

Comment:
I'd adjust the description such that it is Axon Framework specific.
So, something like "Migration file to upgrade from an Axon Framework Javax-specific project to Jakarta".

Reply:
Updated the description for both:
```yaml
description: Migration file to upgrade an Axon Framework Javax-specific project and remain on Javax.
description: Migration file to upgrade from an Axon Framework Javax-specific project to Jakarta.
```